legislators-current.yaml

```diff
@@ -39594,7 +39594,7 @@
     class: 2
     state_rank: junior
     party: Republican
-    url: http://www.rounds.senate.gov
+    url: https://www.rounds.senate.gov
```

Comment:
@joelcollinsdc Did the data itself get updated in this PR? Noting a couple of substantive changes.